ajax/libs/cartodb.js/packages.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "cartodb.js",
+  "filename": "carto.min.js",
+  "description": "CARTO.js is a JavaScript library to create custom location intelligence applications that leverage the power of CARTO. It is the library that powers Builder and it is part of the Engine ecosystem.",
+  "keywords": [
+    "cartodb", 
+    "Maps", 
+    "SQL"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/CartoDB/carto.js.git"
+  },
+  "homepage": "https://cartodb.com/",
+  "author": {
+    "name": "CartoDB",
+    "email": "wadus@cartodb.com"
+  },
+  "license": "BSD-3-Clause",
+  "npmName": "@carto/carto.js",
+  "npmFileMap": [
+    {
+      "basePath": "@carto/carto.js",
```

```suggestion
      "basePath": "",
```

We start from within the package already?